### PR TITLE
release canvas 1

### PR DESCRIPTION
**Pull request by mredig**

### CODE REVIEW

File: swaap/swaap/Helpers/Operations/ConcurrentOperation.swift [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    enum State: String {
+        case isReady, isExecuting, isFinished
+    }
+    
+    // MARK: Properties
+	let task: () -> Void
+    
+    private var _state = State.isReady
+    
+    private let stateQueue = DispatchQueue(label: "com.LambdaSchool.Astronomy.ConcurrentOperationStateQueue")
```
Comment (mredig) [inline swaap/swaap/Helpers/Operations/ConcurrentOperation.swift]: let's change this to a more personal label for ourselves

File: swaap/swaap/Model Controllers/ProfileController.swift [hunk 2]
```diff
@@ -316,6 +314,27 @@ class ProfileController {
 		}
 	}
 
+
+	// MARK: - Image
+	let cloudinaryConfig = CLDConfiguration(cloudName: "swaap", secure: true)
+	lazy var cloudinary = CLDCloudinary(configuration: cloudinaryConfig)
```
Comment (mredig) [inline swaap/swaap/Model Controllers/ProfileController.swift]: these should probably be private

File: swaap/swaap/Model Controllers/ProfileController.swift [hunk 3]
```diff
@@ -347,7 +366,7 @@ class ProfileController {
 
 	private func saveProfileToCache() {
 		guard let profile = userProfile else {
-			deleteProfileCache()
+//			deleteProfileCache()
```
Comment (mredig) [inline swaap/swaap/Model Controllers/ProfileController.swift]: delete, not just comment

File: swaap/swaap/View Controllers/QRViewController.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		let finalImage = UIImage(cgImage: cgimage)
+
+		qrImageView.image = finalImage
+	}
+
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
+		updateViews()
+	}
+}
```
Comment (mredig) [inline swaap/swaap/View Controllers/QRViewController.swift]: this is crap, but it worked

File: swaap/swaap/Views/ProfileCardView.swift [hunk 5]
```diff
@@ -13,6 +13,53 @@ import ChevronAnimatable
...
+	var industry: String? {
+		get { industryLabel.text }
+		set { industryLabel.text = newValue }
+	}
+
+	var preferredContact: SocialLink? {
+		get { socialButton.socialInfo }
+		set {
+			guard let newValue = newValue else { return }
+			socialButton.socialInfo = newValue
```
Comment (mredig) [inline swaap/swaap/Views/ProfileCardView.swift]: why is unwrapping necessary?

Comment (marlonjames71) [reply swaap/swaap/Views/ProfileCardView.swift]: The compiler was telling me it needs to be unwrapped

**Pull request merged**